onnx_xla/translations/translate_unsqueeze.cc

```diff
@@ -0,0 +1,42 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+// Translate Unsqueeze by using XLA's Reshape
+onnxStatus translateUnsqueeze(const Node& n,
+                              XlaBuilder& builder,
+                              ValueOpMap& valueToOp) {
+  // Set origShape and axes
+  std::vector<int64> origShape = OperatorRegistry::parseOnnxInputSizes(n, 0);
+
+  if (!n.hasAttribute(kaxes)) {  // TODO ENFORCE
+    std::cerr << "Missing Required Attribute" << std::endl;
+    return ONNXIFI_STATUS_INVALID_MODEL;
+  }
+  const auto& axes = n.is(kaxes);
+
+  // Loop through both origShape and axes to make newShape
+  auto origIndex = 0;
+  auto axesIndex = 0;
+  std::vector<int64> newShape;
+  while (origIndex < origShape.size() && axesIndex < axes.size()) {
```

Comment:
Looks like from the spec axes are not guaranteed to be in sorted order, I feel the following implementation is simpler:
```c++
std::vector<int64_t> newShape(origShape.size() + axes.size());
for (auto axis : axes) {
  newShape[axis] = 1;
}
for (int i = 0, j = 0; i < newShape.size(); ++i) {
  if (newShape[i] != 1) {
    newShape[i] = origShape[j++];
  }
}
```

Reply:
Oops - if they arent in sorted order this makes sense